spec/std/hash_spec.cr

```diff
@@ -88,6 +88,53 @@ describe "Hash" do
     end
   end
 
+  context "subset/superset operators" do
+    h1 = {"a" => 1, "b" => 2}
```

Comment:
IMHO these variables put to much distance between the values and the tests. I'd suggest to simply use the values directly as literals instead of reusing variables.

Reply:
Distance is not that big and I specifically moved them to the `context` scope since they're being reused by each and every test below so I don't see a reason to duplicate them.